src/main/resources/schemas/magistrates_public_list.json

```diff
@@ -306,12 +306,6 @@
                 "title": "Court House Contact",
                 "$ref": "#/definitions/venueContact"
               },
-              "courtHouseAddress": {
```

Comment:
johTitle also needs removing from this file

Reply:
Removed johTitle from schema -  magistrates_public_list.json